include/dmlc/omp.h

```diff
@@ -5,15 +5,23 @@
  */
 #ifndef DMLC_OMP_H_
 #define DMLC_OMP_H_
+
+
 #if defined(_OPENMP)
 #include <omp.h>
 #else
+
+#if defined(__ANDROID__) && defined(__AARCH64EL__)
+#define __GOMP_NOTHROW
+#elif defined(__cplusplus)
+#define __GOMP_NOTHROW throw()
```

Comment:
Maybe noexcept instead?

Reply:
This was a fix by somebody else, not sure if it can cause problems if we change to noexcept. I would leave it as it is.